src/interfaces/swig/shogun.i

```diff
@@ -403,8 +403,8 @@ PUT_ADD(LossFunction)
 PUT_ADD(StructuredModel)
 PUT_ADD(FactorType)
 
-%template(kernel) kernel<float64_t, float64_t>;
-%template(features) features<float64_t>;
+%template(create_kernel) create_kernel<float64_t, float64_t>;
+%template(create_features) create_features<float64_t>;
```

Comment:
@vigsterkr @karlnapf would it make sense to move these to factory.i? Or is it required to be defined here in SWIG? Same with all the `PUT_ADD`..

Reply:
yep move them. Think I tried that and had problems but I forgot the details :)